helm/hpcc/docs/placements.md

```diff
@@ -20,6 +20,8 @@ The list item in "pods" can be one of the following:
 4) Job name regular expression:  For example "compile-" or "compile-.*" or exact match "^compile-.*$"
 5) set pod: ["all"] for all HPCC Systems components
 
+No mattern what order a user inputs the placements will be processed as "all", "type", "target" and "pod" or "job".
```

Comment:
typo: mattern->matter

Reply:
I'd reword as:

Regardless of the order of placements in the configuration, they will be processed in the following order:  "all", "type", "target" and "pod" or "job".

Reply:
Fixed